security/msal/MsalClient.ts

```diff
@@ -279,7 +280,9 @@ export class MsalClient extends BaseOAuthClient<MsalClientConfig, MsalTokenSpec>
                 try {
                     const {events} = this.telemetryResults,
                         {name, startTimeMs, durationMs, success, errorName, errorCode} = e,
-                        eTime = startTimeMs ? new Date(startTimeMs) : new Date();
+                        eTime = fmtDate(startTimeMs ?? Date.now(), {
+                            fmt: 'MMM D h:mm:ssa'
```

Comment:
Why not just use millis?  Don't think we would want to bake in pre-formatted dates (in browser's timezone)